src/styles/index.css

```diff
@@ -5,6 +5,11 @@
 /* App styles */
 @import './nav.css';
 
+html,
+body {
+  height: 100%;
+}
```

Comment:
Should it be only on mobile ?

Reply:
It's mainly here for mobile purpose, but it doesn't affect desktop. Do you have any issue in mind?